.gitlab/corona-jobs.yml

```diff
@@ -27,11 +27,22 @@ corona_clang_hip:
 # ------------------------------------------------------------------------------
 # HIP + TPLs
 # ------------------------------------------------------------------------------
-corona_clang_hip_tpls:
+corona_rocmcc_550_tpls:
```

Comment:
Looking at the test output, it looks like this doesn't run any of tests with the TPLs.